site/gatsby-site/playwright/e2e-full/unit/AlgoliaUpdater.spec.ts

```diff
@@ -0,0 +1,87 @@
+import { expect } from '@playwright/test';
+import sinon from 'sinon';
+import { ObjectId } from 'bson';
+import AlgoliaUpdater from '../../../src/utils/AlgoliaUpdater';
+import { test } from '../../utils';
+import { MongoClient } from 'mongodb';
+import config from '../../config';
+import { init } from '../../memory-mongo';
+
+test('Should update translations to Algolia', async ({ page }) => {
+
+  init();
+
+  const reporter = { log: sinon.stub() };
+
+  const mongoClient = new MongoClient(config.MONGODB_CONNECTION_STRING);
```

Comment:
hmm, I'm not sure this is how we want to "connect" to the in-memory MongoDB instance. I suspect that the test fixture should expose a client.

The client connect logic for the in-memory instance is handled here, and we probably shouldn't let different connection logic leak:
https://github.com/responsible-ai-collaborative/aiid/blob/4b5603012c380a6beac122717d4b6b48b2df5218/site/gatsby-site/playwright/memory-mongo.ts#L85

@cesarvarela should we expose a client in the test fixture?

Reply:
@cesarvarela I went back with using the mocks, but I didn't use jest and moved to `/server` as there wasn't an easy way to stub the AlgoliaUpdater with it.